apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/InputValidator.java

```diff
@@ -24,9 +24,9 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class InputValidator {
-  public static final String INVALID_CLUSTER_NAMESPACE_MESSAGE = "Only digits, alphabets and symbol - _ . are allowed";
+  public static final String INVALID_CLUSTER_NAMESPACE_MESSAGE = "Only digits, alphabets and symbol - _ . (except single .) are allowed";
   public static final String INVALID_NAMESPACE_NAMESPACE_MESSAGE = "not allowed to end with .json, .yml, .yaml, .xml, .properties";
-  public static final String CLUSTER_NAMESPACE_VALIDATOR = "[0-9a-zA-Z_.-]+";
+  public static final String CLUSTER_NAMESPACE_VALIDATOR = "[0-9a-zA-Z-]+[0-9a-zA-Z._-]*";
```

Comment:
Please add some unit test in [InputValidatorTest](https://github.com/apolloconfig/apollo/blob/master/apollo-common/src/test/java/com/ctrip/framework/apollo/common/utils/InputValidatorTest.java).